docs/faq.rst

```diff
@@ -85,8 +85,15 @@ in the repository.
 How do I set the "Playing" status?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There is a method for this under :class:`Client` called :meth:`Client.change_presence`.
-The relevant aspect of this is its ``activity`` keyword argument which takes in an :class:`Activity` object.
+The ``activity`` keyword argument may be passed in the :class:`Client` constructor, given an :class:`Activity` object.
+
+There is also a method for this called :meth:`Client.change_presence`.
+
+.. warning::
+
+    It is highly discouraged to use :meth:`Client.change_presence` or API calls in :func:`on_ready` as this event may be called many times while running, not just once.
+
+    Discord has a high chance to completely disconnect you during the ``READY`` or ``GUILD_CREATE`` events (``1006`` close code) and there is nothing you can do to prevent it.
```

Comment:
```suggestion
    There is a high chance of disconnecting if presences are changed right after connecting.
```

I want to keep specifics and theory guessing out of it. I don't actually know why it disconnects you (I've tried figuring it out to no avail).